util/bloom.cc

```diff
@@ -181,8 +181,37 @@ class FullFilterBitsReader : public FilterBitsReader {
     // Other Error params, including a broken filter, regarded as match
     if (num_probes_ == 0 || num_lines_ == 0) return true;
     uint32_t hash = BloomHash(entry);
-    return HashMayMatch(hash, Slice(data_, data_len_),
-                        num_probes_, num_lines_);
+    uint32_t bit_offset;
+    FilterPrepare(hash, Slice(data_, data_len_), num_lines_, &bit_offset);
+    return HashMayMatch(hash, Slice(data_, data_len_), num_probes_, bit_offset);
+  }
+
+  virtual void MayMatch(Slice** keys, bool* may_match, int num_keys) override {
```

Comment:
nit: output argument is preferred to be after input arguments: https://google.github.io/styleguide/cppguide.html#Output_Parameters